src/app/shell/admin-tools/data/import-export/import-providers/import-providers.component.html

```diff
@@ -0,0 +1,175 @@
+      {{ 'IMPORT/EXPORT.SUBMIT_FILE' | translate }}
+    </button>
+  </div>
+  <div class="errors-menu">
+    <div class="length-warning" *ngIf="isWarningVisible">
+      <mat-card-subtitle>{{'IMPORT/EXPORT.HUNDRED_WARNING' | translate}}</mat-card-subtitle>
+    </div>
+    <ng-container *ngIf="dataSourceInvalid?.length">
+      <mat-slide-toggle [(ngModel)]="isToggle" color="warn"
+        ><span class="incorrect-row">{{ 'IMPORT/EXPORT.INCORRECT_ROWS' | translate }}</span
+        ><span class="incorrect-row-number">{{dataSourceInvalid.length}}</span></mat-slide-toggle>
+    </ng-container>
+    <app-info-menu *ngIf="dataSourceInvalid?.length" style="font-size: 20px; margin-bottom: 10px; display: flex" [isBigIcon]="true">
+      <div class="errors-types">
+        <h3>{{ 'IMPORT/EXPORT.ERRORS.ERRORS_TYPES' | translate }}</h3>
+        <ul>
+          <li>
+            <div class="color-box error-field-duplicate"></div>
+            <p>{{ 'IMPORT/EXPORT.ERRORS.DATA_DUPLICATE' | translate }}</p>
+          </li>
+          <li>
+            <div class="color-box error-field-empty"></div>
+            <p>{{ 'IMPORT/EXPORT.ERRORS.MISSING_INFO_EMPTY_FIELD' | translate }}</p>
+          </li>
+          <li>
+            <div class="color-box error-field-wrong-length"></div>
+            <p>{{ 'IMPORT/EXPORT.ERRORS.WRONG_LENGTH' | translate }}</p>
+          </li>
+          <li>
+            <div class="color-box error-field-wrong-format"></div>
+            <p>{{ 'IMPORT/EXPORT.ERRORS.WRONG_FORMAT' | translate }}</p>
+          </li>
+          <li>
+            <div class="color-box error-field-wrong-language"></div>
+            <p>{{ 'IMPORT/EXPORT.ERRORS.WRONG_LANGUAGE' | translate }}</p>
+          </li>
+        </ul>
+      </div>
+    </app-info-menu>
+  </div>
+</div>
+
+<ng-container *ngIf="isToggle ? dataSourceInvalid : dataSource">
+  <div class="wrapper-provider-list">
+    <div class="wrapper-provider-list-container">
+      <div class="table-container mat-elevation-z8">
+        <table mat-table [dataSource]="isToggle ? dataSourceInvalid : dataSource" class="custom-table-style">
+          <ng-container matColumnDef="sequence number">
+            <th mat-header-cell *matHeaderCellDef>№</th>
+            <td mat-cell style="min-width: 30px;" *matCellDef="let element">{{ dataSource.indexOf(element) + 1 }}</td>
+          </ng-container>
+          <ng-container matColumnDef="provider name">
+            <th mat-header-cell *matHeaderCellDef>{{ 'IMPORT/EXPORT.PROVIDERS_LIST.PROVIDER_NAME' | translate }}</th>
+            <td
+              mat-cell
+              *matCellDef="let element"
+              [ngClass]="{
+                'error-field-empty': element.errors.providerNameEmpty,
+                'error-field-wrong-length': element.errors.providerNameLength
+              }"
+              [matTooltip]="element.providerName"
+              [showTooltipIfTruncated]
```

Comment:
You don't need to put square brackets here because we're not passing any data to directive

Reply:
Same below